src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/AssetsComputingHelper.cs

```diff
@@ -94,12 +94,29 @@ private static bool IsFromMonoPackage(ITaskItem candidate)
 
     public static string GetCandidateRelativePath(ITaskItem candidate)
     {
+        const string optionalFingerprint = "#[.{fingerprint}]?";
+        const string requiredFingerprint = "#[.{fingerprint}]!";
+
+        string fileName = candidate.GetMetadata("FileName");
+        string extension = candidate.GetMetadata("Extension");
+        string subPath = string.Empty;
+
         var destinationSubPath = candidate.GetMetadata("DestinationSubPath");
```

Comment:
Do we use this anywhere? 

Might be some legacy stuff here, so you might be able to simplify this further

Reply:
Tracking in https://github.com/dotnet/runtime/issues/103810